osu.Game/Overlays/Music/PlaylistItem.cs

```diff
@@ -114,24 +117,62 @@ private void recreateText()
             });
         }
 
-        protected override bool OnHover(Framework.Input.InputState state)
+        protected override bool OnHover(InputState state)
         {
             handle.FadeIn(fade_duration);
 
             return base.OnHover(state);
         }
 
-        protected override void OnHoverLost(Framework.Input.InputState state)
+        protected override void OnHoverLost(InputState state)
         {
             handle.FadeOut(fade_duration);
         }
 
-        protected override bool OnClick(Framework.Input.InputState state)
+        protected override bool OnClick(InputState state)
         {
             OnSelect?.Invoke(BeatmapSetInfo);
             return true;
         }
 
+        protected override bool OnDragStart(InputState state)
+        {
+            return true;
+        }
+
+        // Maybe render some ghost text
+        protected override bool OnDrag(InputState state)
+        {
+            return true;
+        }
+
+        protected override bool OnDragEnd(InputState state)
+        {
+            int src = (int) Depth;
+            int dst = clamp((int) ((state.Mouse.Position.Y - Parent.DrawPosition.Y) / Height), 0, playlist.Count - 1);

```

Comment:
`MathHelper.Clamp`?

Reply:
And no spacing between cast and the variables.

Reply:
Also DrawPosition in this case isn't needed because Mouse.Position.Y is already relative to us.

Reply:
I'm pretty sure the DrawPosition is not 0 and this works, so removing it wouldn't work, but I will try it without to make sure.

Reply:
Turns out the DrawPosition is 0, so removed.